app/uk/gov/hmrc/customs/notification/controllers/HeaderValidator.scala

```diff
@@ -126,9 +130,8 @@ trait HeaderValidator {
     logValidationResult(AUTHORIZATION, result)(h)
   }
 
-  private def logValidationResult(headerName: => String, validationResult: => Boolean)(implicit h: Headers) = {
-    val resultText = if (validationResult) "passed" else "failed"
-    val msg = s"$headerName header $resultText validation"
+  private def logValidationResult(headerName: => String, validationResult: => Boolean, validText: => String = validText, invalidText: String = invalidText)(implicit h: Headers) = {
+    val msg = if (validationResult) s"$headerName $validText" else s"$headerName $invalidText"
```

Comment:
Why don't you just construct the msg inside the if statement below? This one seems redundant